src/api.php

```diff
@@ -162,8 +168,9 @@ function notificationAdd(Notification $notification)
  * @param array<mixed> $data Notification static data.
  * @return array<mixed>       Converted data.
  * @since  6.0.0
+ * @since [Next] Function lives under BracketSpace\Notifiation namespace.
  */
-function notificationConvertData($data = [])
+function convertData($data = [])
```

Comment:
```suggestion
function convertNotificationData($data = [])
```

Let's make this one specific too